src/routes/search/searchHelpers.ts

```diff
@@ -0,0 +1,23 @@
+// import type { Hit } from "@algolia/client-search";
+// import type { AlgoliaHit, HitHighlightResult } from "instantsearch.js";
+
+export function searchHitsTemplate(hit) {
```

Comment:
this will need to be updated to match the shape of the data from Contentful. Right now, it's based on the `media-sample-data` index hits